### PR TITLE
Fill DB with tags

### PR DESCRIPTION
**Pull request by GGrify**

### CODE REVIEW

File: backend/core/app/src/main/resources/db/common/V3__seed_tags.sql [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+('0bcf72f0-d3ae-4944-b115-21a84f0bcb44', 'SVG'),
+('eaaf1e23-4c4e-4a52-a0c1-57234217e736', 'CVG'),
+('fff4d71f-0312-4e0f-87d6-61354bdb5d21', 'Keras'),
+('a2cf0961-f384-46d3-b586-ea068fd9f439', 'Machine-Learning'),
+('349d36bf-8c1e-476d-a6a9-625eb2673d97', 'Intel'),
+('07fa033b-421e-459f-9246-8c3cda200dfb', 'AMD'),
+('c4a10fd2-8e9f-4621-86b5-428d1280136d', 'IntelliJ-IDEA'),
+('89237d9b-add3-4691-a6dd-a626c3ec9305', 'JetBrains'),
+('8c5204f1-b99f-4182-b2db-4e118d4985c8', 'SciPy'),
+('739ba4ce-a0d6-4e11-aa63-551388b2f608', 'pandas');
```
Comment (kopot-o-d) [inline backend/core/app/src/main/resources/db/common/V3__seed_tags.sql:114]: pandas?

Comment (GGrify) [reply backend/core/app/src/main/resources/db/common/V3__seed_tags.sql:114]: pandas - Python Data Analysis Library
https://ru.wikipedia.org/wiki/Pandas

Comment (kopot-o-d) [reply backend/core/app/src/main/resources/db/common/V3__seed_tags.sql:114]: remove pandas

Comment (kopot-o-d) [reply backend/core/app/src/main/resources/db/common/V3__seed_tags.sql:114]: don't use '-' and '.' - just CamelCase

File: backend/core/app/src/main/resources/db/common/V3__seed_tags.sql [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+('4df9cf43-0144-4892-a5da-ac858ff28145', 'Design'),
+('e6427bca-e82d-480b-8eaa-1aec084afce9', 'Electronics'),
+('7b39eaf4-1157-4e8f-9d0a-8471304369a0', 'CyberSecurity'),
+('b98a7973-08cd-4308-a6cb-e8dc4d88e8f6', 'ComputerScience'),
+('a9bd986b-aabd-433e-9d00-8255ad113198', 'Programming'),
+('253cd787-5115-42e4-85c6-e7d0daaa1a56', 'Coding');
+
+
+
+
```
Comment (Quipex) [inline backend/core/app/src/main/resources/db/common/V3__seed_tags.sql:148]: remove empty lines

**Review by Quipex — APPROVED** [review]
lgtm

**Pull request merged**